hoot-core/src/main/cpp/hoot/core/io/OsmApiChangeset.cpp

```diff
@@ -591,11 +708,13 @@ size_t XmlChangeset::getObjectCount(ChangesetInfoPtr& changeset, XmlRelation* re
 
 bool XmlChangeset::isSent(XmlElement* element)
 {
-  //  Sent means Finalized
   if (element == NULL)
     return false;
   else
-    return element->getStatus() == XmlElement::ElementStatus::Finalized;
+    //  Sent means Buffering, Sent, or Finalized
+    return element->getStatus() == XmlElement::ElementStatus::Buffering ||
+           element->getStatus() == XmlElement::ElementStatus::Sent ||
+           element->getStatus() == XmlElement::ElementStatus::Finalized;
```

Comment:
`isSent` tells whether Hootenanny can send an element to the API in the next changeset.  This now will only return true if it isn't sent (Finalized) or already buffered to send (Buffering) or already sent but not returned from the API.